lib/state/appActions.js

```diff
@@ -42,6 +43,29 @@ appActions.trackedState = {
   SEE_IN_BLIP : 'Clicked See Data in Blip'
 };
 
+appActions.errorText = {
+  E_READING_FILE : 'Error reading file: ',
+  E_WRONG_FILE_EXT : 'Please choose a file ending in: ',
+  E_UPLOAD_IN_PROGRESS : 'Cannot start upload while an upload is in progress',
+  E_UNSUPPORTED_TYPE : 'Unsupported upload source type: ',
+  E_INVAILD_UPLOAD_INDEX : 'Invalid upload index: ',
+  E_DEVICE_NOT_CONNECTED : 'The device doesn\'t appear to be connected'
+};
+
+appActions.errorStage = {
+  STAGE_SETUP : { Code: 'E_SETUP' , Message: 'Error during setup', Step: '' },
+  STAGE_LOGIN : { Code: 'E_LOGIN' , Message: 'Error during login', Step: '' },
+  STAGE_LOGOUT : { Code: 'E_LOGOUT' , Message: 'Error during logout', Step: '' },
+  STAGE_DETECT_DEVICES : { Code: 'E_DETECTING_DEVICES' , Message: 'Error detecting devices', Step: '' },
```

Comment:
True @jh-bate, not the objective of this PR.

And yes @kentquirk I guess I meant more that we could go back and find it to look at for reference, not use it directly. I did that when reimplementing the file selection and reading for OmniPod files, but I will admit it took me a fairly long hunt one evening on GitHub to go back far enough in the history to find where that file selection had last existed...